### DISCUSSION TITLE
如何在CommunityToolkit中使用导航？

### DISCUSSION THREAD

**Discussion by hellodoger** [BYJRK/DotNet-Discussions#105] [WPF 开发]
以前用prism框架，自带导航功能，现在尝试CommunityToolkit，突然整不会了，有没有类似的导航框架，切换页面功能，适配CommunityToolkit的？

**Comment by SagapoSheepL** [comment]
我是用了一个叫WPFUI的包，不过看起来没什么人用，而且文档不多，不过开源。

**Comment by LeoChen98** [comment]
我用Frame手工实现，反正控制vm销毁与否和是否单例就好了，窗口/控件/页面都不重要
